Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.yml

```diff
@@ -1970,6 +1970,45 @@ script:
     - contextPath: ServiceNow.Ticket.Note.Type
       description: Ticket note type
       type: string
+  - arguments:
+    - default: false
+      description: Ticket System ID
+      isArray: false
+      name: id
+      required: true
+      secret: false
+    - default: false
+      description: 'Tag system ID. Can be retrieved using: "!servicenow-query-table
+        table_name=label fields=name,active"'
+      isArray: false
+      name: tag_id
+      required: true
+      secret: false
+    - default: false
+      description: tag_title
```

Comment:
I'm not sure about the title, I guess it can be free text, but from the examples I saw it's the incident number like: `Incident - INC000001` etc.